Sources/SimilaritySearchKit/Core/Embeddings/Models/NativeContextualEmbeddings.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  NativeContextualEmbeddings.swift
+//  
+//
+//  Created by Zach Nagengast on 10/11/23.
+//
+
+import Foundation
+import NaturalLanguage
+import CoreML
+
+#if canImport(NaturalLanguage.NLContextualEmbedding)
+@available(macOS 14.0, iOS 17.0, *)
```

Comment:
both if and available seem redundant (minor detail)

Reply:
It's actually necessary to compile on macos 13, since github runners don't have macOS 14 yet.